04-Lab/FunctionalSpatialImageProcessing.ipynb

```diff
@@ -0,0 +1,517 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Lab 04: Functional spatial image processing\n",
+    "\n",
+    "The purpose of this lab is to practice registration steps that transform a functional image to higher-resolution anatomical space of the same subject, and then to standard space. We will then learn how to use these transforms to move an atlas-based mask from standard space to the subject's functional space."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Preparation\n",
+    "* Import python tools for plotting 3D images\n",
+    "* Copy images from Lab 03 for our sub-97 example that we've been working with"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib inline\n",
+    "from nilearn import plotting\n",
+    "from nilearn import image"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%bash\n",
+    "# Copy files from previous labs (or download) so it is easier work with them here\n",
+    "\n",
+    "# Files from 03 Lab that include our transforms for T1->MNI\n",
+    "# DECIDE WHETHER YOU WANT TO COPY (cp) OR DOWNLOAD THE FILES NECESSARY FOR THIS LAB\n",
+    "# default\n",
+    "ans=\"dl\"\n",
```

Comment:
I added the option to download the files necessary to complete this lab, as I know a couple people may have accidentally overwritten some the files we are copying over which will lead to extra time debugging.